packages/math-input/src/store/actions.ts

```diff
@@ -0,0 +1,187 @@
+import type {Key} from "../data/keys";
+import type {
+    KeypadConfiguration,
+    KeyHandler,
+    Cursor,
+    ActiveNodesObj,
+    Border,
+} from "../types";
+
+// naming convention: verb + noun
+// the noun should be one of the other properties in the object that's
+// being dispatched
+
+export const DismissKeypadActionType = "DismissKeypad";
+
+export type DismissKeypadAction = {
+    type: typeof DismissKeypadActionType;
+};
+
+export const dismissKeypad = (): DismissKeypadAction => {
+    return {
+        type: DismissKeypadActionType,
+    };
+};
```

Comment:
So I _think_ we can get away without quite so much typing and still get the same level of type safety.  If you inline this string into the `DismissKeypadAction` type like this, I think you can still get all the same benefits.  

```suggestion
export type DismissKeypadAction = {
    type: "DismissKeypad";
};

export const dismissKeypad = (): DismissKeypadAction => {
    // We can use the literal here because the return value of this function 
    // requires that we return it with that type. 
    return {
        type: "DismissKeypad",
    };
};
```

Reply:
Done